calour/heatmap/plotgui_qt5.py

```diff
@@ -55,13 +61,8 @@ def __call__(self):
             self.app_window.raise_()
             # run the event loop
             self.app.exec_()
-        finally:
-            # clean up when the qt app is closed
-            if self.app_window in self.app.references:
-                logger.debug('removing window from app window list')
-                self.app.references.remove(self.app_window)
-            else:
-                logger.debug('window not in app window list. Not removed')
+        except:
```

Comment:
are you able to provide exception type? it is very dangerous without specifying exception type.

why removing the cleanup code after closing qt gui? Oh, I see. it's moved to `fileQuit` which is called by `closeEvent`. where is `closeEvent` used? could you comment it inside the `closeEvent` function?

btw, is this change for Windows OS?

Reply:
closeEvent is the default class method qt5 calls when a window is closed.
The change is for ezcalour.
I used a general except since we don't know what error we'll get - we try to let the program be a little robust and not fail and stop... but maybe we should let it fail if opening the window fails?
added a comments to explain better :)

Reply:
you can fail it once and see what type of exception it is?